onebusaway-android/src/main/java/org/onebusaway/android/ui/social/SocialAppProfile.java

```diff
@@ -0,0 +1,32 @@
+/*
+* Copyright (c) 2017, Microsoft Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.onebusaway.android.ui.social;
+
+import com.microsoft.embeddedsocial.sdk.ui.AppProfile;
+
+import org.onebusaway.android.R;
+
+public class SocialAppProfile implements AppProfile {
+    @Override
+    public int getName() {
+        return R.string.app_name;
+    }
+
+    @Override
+    public int getImage() {
+        return R.mipmap.ic_launcher;
```

Comment:
Where is this icon used? I don't see an obvious use of it. `mipmap` icons should technically only be referenced for the launcher (see https://developer.android.com/studio/write/image-asset-studio.html), and the OBA icon when shown in-app should be pulled from `drawable`.  However, I don't believe we currently display the OBA logo anywhere in-app.  If this logo is shown somewhere where exact dp size doesn't matter, then this is ok as-is.

Reply:
This is shown as the profile picture for OBA created topics for bus stops and routes.